doc/whats_new/v0.24.rst

```diff
@@ -587,6 +587,17 @@ Changelog
   with different endianness.
   :pr:`17644` by :user:`Qi Zhang <qzhang90>`.
 
+:mod:`sklearn.utils`
+....................
+
+- |Fix| In ``fit`` and ``fit_transform`` methods of Spectral bi-clustering 
+  algorithms :class:`SpectralCoclustering` and :class:`SpectralBiclustering`, 
+  estimator :class:`PLSSVD`, and dimensionality reduction algorithms 
```

Comment:
`PLSSVD` should have its own entry because `random_state` was added to it.

I would place the rest in the `Changed models` section at the top because it changes the solution for module models.

Note that the module need to be included for the references to link correctly, i.e.:

```rst
:class:`cluster.SpectralCoclustering`
```